pHome:
  x: 78.751
  y: 262.814
  z: 591.329
  rx: -123.664
  ry: 14.020
  rz: 119.448
  j1: 90.533
  j2: -77.652
  j3: -64.672
  j4: -130.668
  j5: 84.880
  j6: -6.520